pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -287,7 +287,44 @@ func describeProjectAndServer(project *projectapi.Project, server string) string
 	}
 }
 
-func describeDeploymentInServiceGroup(deploy graphview.DeploymentConfigPipeline) []string {
+func describeImageStreamTagRuntimeStatus(imageSpec string, c client.Interface, namespace string) string {
+	segments := strings.Split(imageSpec, "/")
+	streamPlusTag := ""
+	project := ""
+	if len(segments) == 2 {
+		project = segments[0]
+		streamPlusTag = segments[1]
+	} else {
+		streamPlusTag = imageSpec
+	}
```

Comment:
pretty sure there's a helper somewhere for parsing ImageStreamTags already.  @ncdc ?